app/templates/components/payments/funds-recipient/verification-document.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if (eq stripleConnectAccount.verification_document_status 'pending_requirement')}}
+  {{!-- Show nothing --}}
+{{else if (eq stripleConnectAccount.verification_document_status 'required')}}
+  we need you to upload a scan of your ID (form + submit)
+{{else if (eq stripleConnectAccount.verification_document_status 'verifying')}}
+  Please be patient while we review the document you provided.
+{{else if (eq stripleConnectAccount.verification_document_status 'verified')}}
```

Comment:
I would just do `if (eq stripleConnectAccount.verification_document_status 'required')` and `if (eq stripleConnectAccount.verification_document_status 'verifying')` without the branching. Otherwise, it will by default render nothing.